docs/en/API/Auto-API-Controllers.md

```diff
@@ -2,7 +2,7 @@
 
 Once you create an [application service](../Application-Services.md), you generally want to create an API controller to expose this service as an HTTP (REST) API endpoint. A typical API controller does nothing but redirects method calls to the application service and configures the REST API using attributes like [HttpGet], [HttpPost], [Route]... etc.
 
-ABP can **automagically** configure your application services as API Controllers by convention. Most of time you don't care about its detailed configuration, but it's possible to fully customize it.
+ABP can **"automagically"** configure your application services as API Controllers by convention. Most of time you don't care about its detailed configuration, but it's possible to fully customize it.
```

Comment:
The double quotes here don't seem necessary in our case.

Reply:
Automagically isn't a word. I put it in quotes since it's more of an expression than a word. But sure, I'll remove the quotes.

ℹ️ In the future, you might want to use the suggestion button while reviewing. That lets you suggest the change and then I can one-click tap to apply it. Might save you time going forward:

> ![image](https://user-images.githubusercontent.com/2148318/197776679-004a69c7-683c-46ee-b291-c68729723c00.png)

Reply:
✅ This is now complete